dist/Dockerfile.deploy/Dockerfile

```
@@ -1,8 +1,8 @@
 FROM local/cincinnati-build:latest AS builder
 # build
 COPY . .
-RUN cargo build --release && \
-    mkdir -p /opt/cincinnati/bin && \
+RUN if [ ! -z target/release/graph-builder ]; then cargo build --release; fi
```

How does

```
-z STRING
       the length of STRING is zero
 ``` 
work here? I'd expect 
```
-f FILE
    FILE exists and is a regular file
```

_(both are from `man test`)_

So maybe

```suggestion
RUN [ -f target/release/graph-builder ] || cargo build --release
```
?